api/StreamGetApi.h

```diff
@@ -64,7 +64,12 @@ class  StreamGetApi {
     /// <param name="path">irods absolute logical path to the file</param>
     /// <param name="offset">number of bytes to skip for pagination (optional)</param>
     /// <param name="limit">maximum number of bytes to return (optional)</param>
```

Comment:
not optional, and switch documentation to match order of call.

Reply:
done